tilequeue/metatile.py

```diff
@@ -0,0 +1,51 @@
+import zipfile
+import StringIO
+from collections import defaultdict
+from tilequeue.format import zip_format
+
+
+def make_single_metatile(size, tiles):
+    """
+    Make a single metatile from a list of tiles all having the same
+    coordinate and layer.
+    """
+
+    assert size == 1, \
+        "Tilequeue only supports metatiles of size one at the moment."
+
+    if len(tiles) == 0:
+        return []
+
+    coord = tiles[0]['coord']
+    layer = tiles[0]['layer']
+
+    buf = StringIO.StringIO()
+    with zipfile.ZipFile(buf, mode='w') as z:
+        for tile in tiles:
+            assert tile['coord'] == coord
+            assert tile['layer'] == layer
+
+            tile_name = '0/0/0.%s' % tile['format'].extension
```

Comment:
Yup. I thought about whether we'd want it absolute or relative, and the only real difference is if we want "relocatable" metatiles. This might be useful if, for example, we have an "sea" one containing only an ocean rectangle. I couldn't see any downsides to making it relative, but I could easily be overlooking something. What do you think?

I hope we can support metatiling in `x`, `y` and possibly even `z` directions in a future version. But since none of these are supported in this version, I figured it was safe to hard-code this to `0/0/0`. :wink: